views/stripe/subscriptions.view.lkml

```diff
@@ -120,6 +120,11 @@ view: subscriptions {
     sql: ${TABLE}."QUANTITY" ;;
   }
 
+  dimension: plan {
+    type: string
+    sql: COALESCE(${TABLE}."PLAN":product, ${TABLE}."METADATA":current_product_id) ;;
```

Comment:
What Jira ticket is this associated with?

Reply:
this isn't associated to a Jira ticket, the information this field is bringing is already available in product_id, but that field is missing some data. I'd like to use this field over product_id.

Reply:
Thanks @sri-byte, that works :+1:  Let's add it to the PR description so others have the context on the changes (if they happen to look at the PR at a later date).